src/pages/resultsView/enrichments/CopyNumberEnrichmentsTab.tsx

```diff
@@ -22,24 +23,28 @@ export default class CopyNumberEnrichmentsTab extends React.Component<ICopyNumbe
     private uniqueColorGetter = makeUniqueColorGetter();
 
     @autobind
-    private onProfileChange(molecularProfile: MolecularProfile) {
-        this.props.store._selectedEnrichmentCopyNumberProfile = molecularProfile;
+    private onProfileChange(profileSet:{[id:string]:MolecularProfile}) {
+        this.props.store.setCopyNumberEnrichmentProfileSet(profileSet);
     }
 
     public render() {
 
-        switch (getMobxPromiseGroupStatus(this.props.store.copyNumberHomdelEnrichmentData, this.props.store.copyNumberAmpEnrichmentData)) {
+        switch (getMobxPromiseGroupStatus(this.props.store.studies, this.props.store.selectedStudyCopyNumberEnrichmentProfileSet, this.props.store.copyNumberHomdelEnrichmentData, this.props.store.copyNumberAmpEnrichmentData)) {
             case "pending":
                 return <Loader isLoading={true} center={true} size={"big"} />;
             case "error":
                 return <ErrorMessage/>;
             case "complete":
-            default:
+            default: {
```

Comment:
i think you should avoid using curly brackets here if it's just for readability.  not sure what support for this is yet.  it could be useful for block scope, but you are not actually declaring anything.  if you do return, then the () will serve same organlization purpose.